python/dask_cudf/dask_cudf/batcher_sortnet.py

```diff
@@ -137,3 +146,164 @@ def sort_delayed_frame(parts, by):
     valid = compute(valid_ct)[0]
     validparts = parts[:valid]
     return validparts
+
+
+def set_partitions_hash(df, columns, npartitions):
+    c = hash_object_dispatch(df[columns], index=False)
+    return np.mod(c, npartitions)
+
+
+def _shuffle_group_2(df, columns, ignore_index):
+    if not len(df):
+        return {}, df
+    ind = hash_object_dispatch(df[columns], index=False)
+    n = ind.max() + 1
+    result2 = group_split_dispatch(
+        df, ind.values.view(), n, ignore_index=ignore_index
```

Comment:
What is `.view()` here? I'm assuming `ind` is a Series which means `.values` is a cupy array.

Reply:
Right - `ind.values` is a cupy array.  Dask uses `ind.values.view(np.int64)` - Not sure if it makes sense to use `view()` at all here?

Reply:
Yea if we're not needing to translate the dtype I'd suggest we just drop the `.view()` call